backend/services/services.py

```
@@ -0,0 +1,213 @@
+from database.session import SessionLocal
+from zoneinfo import ZoneInfo
+from sqlalchemy.orm import Session, joinedload
+import shutil
+from core.connection_crud import get_connections
+from crud.notification import create_notification
+from AI.moderation import moderate_text
+
+STATUS_404_ERROR = "Post not found"
+
+# Helper function to get the latest post after the provided `last_seen_post`
+def get_newer_posts(last_seen_post: Optional[int], db: Session):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_newer_posts`

(ANN201)